pkg/pillar/cmd/upgradeconverter/upgradeconverter.go

```diff
@@ -126,3 +135,25 @@ type ConversionHandler struct {
 	description string
 	handlerFunc HandlerFunc
 }
+
+//RunPostVaultHandlers invokes postVaultconversionHandlers
+//and notifies the caller through the provided ucChan channel
+//the channel is useful for calling from other agent modules
+//without missing watchdog, by spawning this as a task, and
+//select()ing for its completion
+func RunPostVaultHandlers(ps *pubsub.PubSub,
+	loggerArg *logrus.Logger,
+	logArg *base.LogObject,
+	debugOverride bool, ucChan chan<- int) {
+	logger = loggerArg
```

Comment:
Do we want to log this as source=vaultmgr or source=upgradeconverter?

Reply:
Updated it to take moduleName as an argument